app/src/main/java/com/example/app/baseballmessenger/Card.java

```diff
@@ -154,12 +163,13 @@ public Card(String uuid, String owner, String name, String cond, int num, String
      * @param inCollection Where the card is located
      * @param uuid The uuid of the card to retrieve
      */
-    public Card(String user, boolean inCollection, String uuid){
+    public Card(String user, boolean inCollection, String uuid, final MyCallback myCallback){
```

Comment:
Not sure if this callback is required anymore because the bug it fixed might have been caused by the bad indices in Card, will just leave it in here for now